src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react'
+
+import classNames from 'classnames'
+
+import fallbackCard from '@/assets/fallback_card.jpg?url'
+import * as Images from '@/assets/images'
+
+import Button from '../Button'
+import Sticker from '../Sticker'
+import styles from './GameInfoV2.module.scss'
+
+interface GameImageProps {
+  src?: string
+  alt?: string
+  fallback?: React.ReactNode
+  className?: string
+}
+
+interface EditorChoiceType {
+  isEditorChoice: boolean
+  year?: number
+  className?: string
+}
+
+interface SocialLinks {
+  IconButton: React.ElementType
+  url: string
+}
+
+interface GameInfoV2i18n {
+  title: string
+  version: string
+  earlyAccess: string
+  playerCount: string
+  addingText?: string
+  notAddedToLibrary?: string
+  playText?: string
+  blockchainLabel?: string
+}
+
+interface BlockchainIconProps {
+  icon: React.ElementType
+}
+
+export interface GameInfoV2Props {
+  title: string
+  version?: string
+  earlyAccess?: boolean
+  playerCount?: string
+  image?: {
+    src: string
+    alt: string
+    fallback?: React.ReactNode
+  }
+  blockchains?: Array<{
+    icon: React.ElementType
+    isHoverable?: boolean
+    name?: string
+  }>
+  showRemainingCount?: boolean
+  remainingCount?: string | number
+  socialLinks?: SocialLinks[]
+  editorChoice?: EditorChoiceType
+  actionButton?: React.ReactNode
+  className?: string
+  i18n?: GameInfoV2i18n
+}
+
+const GameImage: React.FC<GameImageProps> = ({
+  src,
+  alt,
+  fallback = (
+    <img
+      src={fallbackCard}
+      alt="Game cover fallback"
+      className={styles.fallbackImage}
+    />
+  ),
+  className
+}) => {
+  const [hasError, setHasError] = React.useState(false)
+
+  if (!src || hasError) {
+    return fallback ? (
+      <div className={classNames(styles.imageFallback, className)}>
+        {fallback}
+      </div>
+    ) : null
+  }
+
+  return (
+    <img
```

Comment:
we should be able to support `next/image` here instead of just `img`. I think we should take the image tag as a prop

Reply:
I don't think it's possible to use next in the UI lib. Could you explain this?